clients/unshaded/src/main/java/tachyon/client/table/RawTableMasterClient.java

```diff
@@ -75,7 +77,8 @@ public synchronized long createRawTable(final TachyonURI path, final int columns
     return retryRPC(new RpcCallableThrowsTachyonTException<Long>() {
       @Override
       public Long call() throws TachyonTException, TException {
-        return mClient.createRawTable(path.getPath(), columns, metadata);
+        RpcOptions rpcOptions = new RpcOptions().setKey(IdUtils.createRpcId());
```

Comment:
That will make things a lot better once we start making all RPCs use IDs, but for now we're only using this in one place and the build is breaking without this change